lib/index.js

```diff
@@ -18,25 +18,24 @@ function inspect(root, targetFile, options) {
   var mvnArgs = buildArgs(root, targetFile, options.args);
   return subProcess.execute('mvn', mvnArgs, {cwd: root})
     .then(function (result) {
-      try {
-        var parseResult = parse(result, options.dev);
-        return {
-          plugin: {
-            name: 'bundled:maven',
-            runtime: 'unknown',
-          },
-          package: parseResult.data,
-        };
-      } catch (error) {
-        error.message = error.message + '\n\n' +
-          'Please make sure that Apache Maven Dependency Plugin ' +
-          'version 2.2 or above is installed, and that ' +
-          '`mvn ' + mvnArgs.join(' ') + '` executes successfully ' +
-          'on this project.\n\n' +
-          'If the problem persists, collect the output of ' +
-          '`mvn ' + mvnArgs.join(' ') + '` and contact support@snyk.io\n';
-        throw error;
-      }
+      var parseResult = parse(result, options.dev);
+      return {
+        plugin: {
+          name: 'bundled:maven',
+          runtime: 'unknown',
+        },
+        package: parseResult.data,
+      };
+    })
+    .catch(function (error) {
```

Comment:
Instead of handling just parsing failures, this clause now catches `subProcess.execute` errors as well.